lib/resources/example_json/r4_examples_financial_transaction.rb

```diff
@@ -131,11 +131,11 @@ module Resources
         },
         {
           'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-alias',
-          'valueString': '1300'
+          'valueString': '0111'
```

Comment:
should line 54 be this new alias too?

Reply:
Reason for not changing this example, It was mentioned that adjustment example should not be changed and remain same in the Jira.

Reply:
Gotcha +1